.github/workflows/run_mmperf.yml

```diff
@@ -0,0 +1,174 @@
+# Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# Workflow for running `mmperf` (https://github.com/mmperf/mmperf).
+# `mmperf` benchmarks matrix-multiply workloads on IREE and other backends such
+# as RUY, TVM, Halide, CuBLAS, etc.
+#
+# The workflow runs benchmarks on CPU and GPU and uploads results to the
+# `mmperf-benchmark-artifacts` GC bucket.
+
+name: mmperf
+
+on:
+  workflow_dispatch:
+  pull_request:
+
+env:
+  GCS_DIR: gs://iree-github-actions-${{ github.event_name == 'pull_request' && 'presubmit' || 'postsubmit' }}-artifacts/${{ github.run_id }}/${{ github.run_attempt }}
+
+jobs:
+  setup:
+    runs-on: ubuntu-20.04
+    env:
+      # The commit being checked out is the merge commit for the PR. Its first
+      # parent will be the tip of main.
+      BASE_REF: HEAD^
+      PR_TITLE: ${{ github.event.pull_request.title }}
+      PR_BODY: ${{ github.event.pull_request.body }}
+      MMPERF_REPO_DIR: "/usr/local/src/mmperf/mmperf"
```

Comment:
This is really a property of the docker container, right? I think it would make more sense to set it in the docker ENV

Reply:
Moved to docker container